src/makeCalculator.js

```diff
@@ -38,6 +38,45 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(number1, number2) {
+      let thisResult = number1;
```

Comment:
why do you create this variables?

Reply:
https://github.com/mate-academy/js_advanced_calculator/pull/1049